azure-iot-device/azure/iot/device/iothub/models/message.py

```diff
@@ -53,6 +54,10 @@ def __init__(
         self.content_encoding = content_encoding
         self.content_type = content_type
         self.output_name = output_name
+        self.iothub_interface_id = None
```

Comment:
>self.iothub_interface_id = None [](start = 8, length = 31)

This attribute should be prefaced with an underscore, i.e. `self._iothub_interface_id` to indicate it should not be altered by the user (unlike the other attributes, which can be altered)

Reply:
Indeed this attribute could be set on a received message, therefore it is defined as a public attribute.

Reply:
Indeed this attribute could be set on a received message, therefore it is defined as a public attribute.